homeassistant/components/icloud/__init__.py

```diff
@@ -112,7 +107,6 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     password = entry.data[CONF_PASSWORD]
     with_family = entry.data[CONF_WITH_FAMILY]
     max_interval = entry.data[CONF_MAX_INTERVAL]
-    gps_accuracy_threshold = entry.data[CONF_GPS_ACCURACY_THRESHOLD]
```

Comment:
This means existing entries still will have this configuration setting possibly. Please add code to clean up existing configuration entries on entry setup.